modules/combined/tests/grain_texture/grain_texture_test_1.i

```diff
@@ -0,0 +1,153 @@
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./var_indices]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[Kernels]
+  # Kernel block, where the kernels defining the residual equations are set up.
+  [./PolycrystalKernel]
+    # Custom action creating all necessary kernels for grain growth.  All input parameters are up in GlobalParams
+  [../]
+[]
+
+[AuxKernels]
+  # AuxKernel block, defining the equations used to calculate the auxvars
+  [./bnds_aux]
+    # AuxKernel that calculates the GB term
+    type = BndsCalcAux
+    variable = bnds
+    execute_on = timestep_end
+  [../]
+  [./unique_grains]
+    type = FeatureFloodCountAux
+    variable = unique_grains
+    execute_on = timestep_end
+    bubble_object = grain_tracker
+    field_display = UNIQUE_REGION
+  [../]
+  [./var_indices]
+    type = FeatureFloodCountAux
+    variable = var_indices
+    execute_on = timestep_end
+    bubble_object = grain_tracker
+    field_display = VARIABLE_COLORING
+  [../]
+[]
+
+[BCs]
+  # Boundary Condition block
+  [./Periodic]
+    [./top_bottom]
+      auto_direction = 'x y' # Makes problem periodic in the x and y directions
+    [../]
+  [../]
+[]
+
+[Materials]
+  [./CuGrGr]
+    # Material properties
+    type = GBEvolution # Quantitative material properties for copper grain growth.  Dimensions are nm and ns
+    block = 0 # Block ID (only one block in this problem)
+    GBmob0 = 2.5e-6 #Mobility prefactor for Cu from Schonfelder1997
+    GBenergy = 0.708 # GB energy in J/m^2
+    Q = 0.23 #Activation energy for grain growth from Schonfelder 1997
+    T = 450 # K   #Constant temperature of the simulation (for mobility calculation)
+    wGB = 14 # nm    #Width of the diffuse GB
+  [../]
+[]
+
+[UserObjects]
+  [./grain_tracker]
+    type = GrainTracker
+    threshold = 0.2
+    connecting_threshold = 0.08
+    flood_entity_type = ELEMENTAL
+    outputs = none
+  [../]
+  [./euler_angle_file]
+    type = EulerAngleFileReader
+    file_name = grn_3_rand_2D.tex
+  [../]
+[]
+
+[VectorPostprocessors]
+  [./textureInfo]
+    type = GrainTextureVectorPostprocessor
+    unique_grains = unique_grains
+    euler_angle_provider = euler_angle_file
+    sort_by = id # sort output by elem id
+  [../]
+[]
+
+[Executioner]
+  type = Transient # Type of executioner, here it is transient with an adaptive time step
+  scheme = bdf2 # Type of time integration (2nd order backward euler), defaults to 1st order backward euler
+
+  #Preconditioned JFNK (default)
+  solve_type = 'PJFNK'
+
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart -mat_mffd_type'
+  petsc_options_value = 'hypre boomeramg 101 ds'
+  l_max_its = 30 # Max number of linear iterations
+  l_tol = 1e-4 # Relative tolerance for linear solves
+  nl_max_its = 40 # Max number of nonlinear iterations
+  nl_abs_tol = 1e-11 # Relative tolerance for nonlinear solves
+  nl_rel_tol = 1e-10 # Absolute tolerance for nonlinear solves
+  start_time = 0.0
+  num_steps = 1.0
```

Comment:
Can we simplify these inputs considerably? Do they need to use hypre? Can we remove options and accept defaults (remove BCs, simplify GrainTracker params.. actually remove GrainTracker, numop=numgrain!)? Can we use a GenericConstantMaterial to set up the properties? I fear that having all this complexity in the input (and that goes for the other input as well) could end up causing problems in the future (the more objects you have in the test the more code changes may influence the test).